cmd/server/main.go

```
@@ -0,0 +1,63 @@
+		api.POST("/leaves", leafHandler.AddLeaf)
+		api.PATCH("/leaves/:id", leafHandler.UpdateLeaf)
+		api.DELETE("/leaves/:id", leafHandler.DeleteLeaf)
+	}
+
+	// Portを環境変数から取得
+	// デフォルトは8080
+	port := os.Getenv("APP_PORT")
+	if port == "" {
+		port = "8080"
+	}
+	r.Run(":" + port)
```

golangci-lint (1.64.8)

Error return value of `r.Run` is not checked

(errcheck)